README.md

```diff
@@ -106,17 +106,17 @@ _The number of blocks that the `hypersdk` stores on-disk, the `AcceptedBlockWind
 to an arbitrary depth (or set to `MaxInt` to keep all blocks). To limit disk IO used to serve blocks over
 the P2P network, `hypervms` can configure `AcceptedBlockWindowCache` to store recent blocks in memory._
 
-### WASM-Based Programs
+### WASM-Based Contracts
```

Comment:
```suggestion
### WASM-Based Smart Contracts
```

i think we should use Smart Contracts in the readme(not for everything but important stuff like title)